File: sdk/samples/all in on code/Visual Studio 2010/VBAzureServiceBusProtocolBridging/Common/IMyService.vb
'****************************** Module Header ******************************\
' Project Name:   CSAzureServiceBusProtocolBridging
' Module Name:    Common
' File Name:      IMyService.vb
' Copyright (c) Microsoft Corporation
'
' A WCF service contract that is shared by both Client project and the Service project.
' 
' This source is subject to the Microsoft Public License.
' See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
' All other rights reserved.
'
'*****************************************************************************/

Imports System.ServiceModel

<ServiceContract()> _
Public Interface IMyService
    <OperationContract()> _
    Function Add(number1 As Integer, number2 As Integer) As Integer
End Interface

File: sdk/samples/all in on code/Visual Studio 2010/CSAzureServiceBusProtocolBridging/Common/IMyService.cs
/****************************** Module Header ******************************\
* Project Name:   CSAzureServiceBusProtocolBridging
* Module Name:    Common
* File Name:      IMyService.cs
* Copyright (c) Microsoft Corporation
*
* A WCF service contract that is shared by both Client project and the Service project.
* 
* This source is subject to the Microsoft Public License.
* See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
* All other rights reserved.
*
\*****************************************************************************/

using System.ServiceModel;

namespace Common
{
    [ServiceContract]
    public interface IMyService
    {
        [OperationContract]
        int Add(int number1, int number2);
    }
}
